### ISSUE TITLE
Unpublished link should copy as "www"

### ISSUE THREAD

**Issue by egseery** [massgov/feature-requests#230]
**Is your feature request related to a problem? Please describe.**
When I need to create a link to an unpublished page, it has always been so that I can send it to someone to view the page, Most often it is someone who doesn't have CMS access. Currently, the link is an "edit." link, so I have to copy and then manually change it to be WWW for my non cms users to use.

**Describe the solution you'd like**
Have the link be www. I have never benefited from it being edit. 

**Additional context**
Add any other context, use cases, or screenshots about the feature request here.

**Reference**
Include JIRA ticket number or ServiceNow ticket numbers that correspond to this issue.

**Comment by viv-lam** [comment]
I agree! It's an extra step that can get cumbersome, especially when creating draft service pages where I link have to link multiple draft pages for the full picture to show program teams.

**Comment by joeg8612** [comment]
implemented
